### DISCUSSION TITLE
Fixed tab on Workspace

### DISCUSSION THREAD

**Discussion by cantalupo555** [Floorp-Projects/Floorp#606] [Feature Request]
When I browse among the workspace, the fixed tabs remain.
Is there any option for all fixed tabs corresponding to your workspace?

![image](https://github.com/Floorp-Projects/Floorp/assets/12485871/f631883f-bd34-4e70-bef1-1b1ac7359318)

**Comment by Ryosuke-Asano** [comment]
I will abandon workspaces future.

I will have the option to migrate to other extensions in the future and will give you 6 months to do so

**Reply by cantalupo555** [reply]
Is there any extension that does something like the workspace?

Why will it be abandoned?

**Reply by Ryosuke-Asano** [reply]
The reason is that the code I wrote is not very good and keeps creating bugs. This is like poison for the user.

I advise users to move to STG in the future.

**Reply by cantalupo555** [reply]
What would STG be?
Will the workspace functionality be rescribed?

**Reply by Ryosuke-Asano** [reply]
This addon: https://addons.mozilla.org/ja/firefox/addon/simple-tab-groups/

All Floorp's Workspaces features covered by it.

Currently, I', creating migration tools for STG. You can carry Workspace icons, Tabs & Containers

**Reply by Ryosuke-Asano** [reply]
Thank you for your sponsor ship!

**Reply by cantalupo555** [reply]
I will use this extension, thanks.
It would be graceful to have something native to Vivaldi.

**Reply by cantalupo555** [reply]
At the moment I'm making money in BRL. 😐
Next year, I will be winning in EUR and so I can increase my contribution to the sponsor, very soon. 😀

**Reply by Ryosuke-Asano** [reply]
Thanks!

**Reply by Ryosuke-Asano** [reply]
I promise, I will create it

**Reply by cantalupo555** [reply]
Thank you very much for implementing version 11.8.0 and receiving the features I requested.
Could you tell me how I can create the Workspace?

**Reply by Ryosuke-Asano** [reply]
Read release note.

You should add button to toolbar via "Customize toolbar"
